packages/@react-spectrum/table/src/TableView.tsx

```diff
@@ -563,11 +563,11 @@ function TableView<T extends object>(props: SpectrumTableProps<T>, ref: DOMRef<H
         <DragPreview ref={preview}>
           {() => {
             let item = state.collection.getItem(dragState.draggedKey);
-            let rowHeaderColumnKeys = state.collection.rowHeaderColumnKeys;
             let itemCount = dragState.draggingKeys.size;
             let maxWidth = bodyRef.current.getBoundingClientRect().width;
-            let children = state.collection.getChildren(item.key);
-            return <SpectrumDragPreview layout={layout} item={item} children={children} itemCount={itemCount} density={density} direction={direction} maxWidth={maxWidth} rowHeaderColumnKeys={rowHeaderColumnKeys} />;
+            let {height} = layout.getLayoutInfo(item.key).rect;
```

Comment:
this looks a little funny in overflow wrap: https://reactspectrum.blob.core.windows.net/reactspectrum/7c01562f85a10dbb0e214e528296927ebc2d040d/storybook/index.html?path=/story/tableview-drag-and-drop--draggable-rows-copy-link&args=overflowMode:wrap&providerSwitcher-express=false

Can we use a standard height and set overflow to truncate for the default preview?

given that it looked funny before, maybe add an item to the backlog as followup?

Reply:
Sure, we could probably just use the `ROW_HEIGHTS` for this, and I agree about truncating overflow.